### DISCUSSION TITLE
Can you please add an option for linux transparency without blur?

### DISCUSSION THREAD

**Discussion by 677230756E64** [QNetITQ/WaveFox#227] [Ideas]
Don't like the blur much and don't use it for other windows, so blur exclusively on firefox window looks odd. If you can, please add an option for simple transparent tab bar, also would be great to be able to choose arbitrary transparency percentage. Values like low, medium, high don't explain much. If I have transparency for the rest of the system set at 90%, I don't know how it matches up with your values.

**Comment by QNetITQ** [comment]
The style does not control the background blur. This is done by a script in the operating system. 

Transparency scripts on Linux allow to control blur, but not the window color. Therefore, I had to set different transparency values ​​​​to use the system color for the background for better visibility of elements. 

The browser does not allow to set any numeric value via keys, only logical true/false. If these scripts allowed to control the window color, I would leave just transparency, like on Windows.
